facebook/__init__.py

```diff
@@ -322,6 +322,23 @@ def extend_access_token(self, app_id, app_secret):
 
         return self.request("oauth/access_token", args=args)
 
+    def debug_access_token(self, token, app_id, app_secret):
+        """
+        Gets information about a user access token issued by an app. See
+        <https://developers.facebook.com/docs/facebook-login/access-tokens
+        #debug>
+
+        We can generate the app access token by concatenating the app
+        id and secret: <https://developers.facebook.com/docs/
+        facebook-login/access-tokens#apptokens>
+
+        """
+        args = {
+            "input_token": token,
+            "access_token": "%s|%s" % (app_id, app_secret)
```

Comment:
Is there a reason why you are concatenating the application ID and secret instead of calling `get_app_access_token`?

I know the former works (and that Facebook's PHP SDK did the same the last time I looked at it), but since [Facebook's official documentation suggests making the OAuth call](https://developers.facebook.com/docs/facebook-login/access-tokens#apptokens), I worry that they might change this behavior in the future (or even for a subset of applications) and that your implementation would break.